docker/prod/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:lts
+
+# Port for web server
+ENV PORT=8080
+EXPOSE 8080
+
+# Install sqlite3 and redis
+RUN apt-get update -qq
+RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -qq sqlite3 git
+
+# Create app directory
+WORKDIR /usr/src/
+
+# Install app dependencies
+RUN git clone https://github.com/dyc3/opentogethertube app 
```

Comment:
Why not just copy the files from outside the container? IIRC that's what you did in the last PR